index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="StudentWEB" content="Social media for the students all around the world to catch up with like-minden fellow coleagues">
+    <title> StudentWEB | Log In</title>
+    <link rel="stylesheet" href="/dist/css/index.css">
+    <link rel="stylesheet" href="/src/scss/index.scss">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.5.0/dist/js/bootstrap.bundle.min.js"></script>
+</head>
+<body class="d-flex flex-column min-vh-100">
+    <header>
+        <nav class="navbar navbar-light bg-light px-2">
+            <h1 class="navbar-brand mb-0 ">StudentWEB</h1>
+        </nav>
+    </header>
+    <main class="d-flex flex-row my-5 mx-5">
+        <div class="container-fluid justify-content-between col-xs-12 col-lg-6 text-center mt-5">
+            <h3>Discover Your Global Network of Like-Minded Students.</h3>
+            <h3>Connect, Explore, and Grow Together.</h3>
+        </div>
+        <div class="container-fluid col-xs-12 col-lg-6">
+            <h3>Sign In</h3>
+                <form action="profile/index.html" class="form-control-md needs-validation">
+                    <div><label for="email" class="form-label">Email address</label><br>
+                        <input type="email" id="email"  placeholder="E-mail" class="d-inline-flex p-2 form-control" required>
+                    </div>
+                    <div class="my-2"><label for="password" class="form-label">Password</label><br>
+                        <input type="password" minlength="8" required placeholder="Password" class="d-inline-flex p-2 form-control">
+                    </div>
+                    <div>
+                        <a href="profile.html"> <button  type="submit" class="btn btn-primary">Log In</button></a>
+                    </div>
+                </form>
+                <hr>
+                <h3>Register</h3>
```

Comment:
Both forms should not be shown at the same time on the page. For this Ca, you can remove register form and stick to login.

Reply:
It is hidden for now. I will put it back with onclick function for JS2.